### PR TITLE
Added Riot.js

### PR DESCRIPTION
**Pull request by crisward**
Really like this framework. Hopefully you can include it in your comparison table. Similar api to react < 10kb. http://riotjs.com/

### CODE REVIEW

File: _data/frameworks.yml [hunk 1]
```diff
@@ -87,11 +94,13 @@
       polymer: <div content$="{{ raw_html }}"></div>
       vue: <div>{{{ raw_html }}}</div>
       ember: <div>{{{ raw_html }}}</div>
+      riot:
```
Comment (jeffcarp) [inline _data/frameworks.yml:97]: Can you add `comment: see notes` here?

File: _data/frameworks.yml [hunk 2]
```diff
@@ -356,6 +394,7 @@
       polymer: created
       vue: created
       ember: init
+      riot: this.on('before-mount',callback)
```
Comment (jeffcarp) [inline _data/frameworks.yml]: To make these cleaner I think I would prefer just raw event names, e.g.

```
riot: before-mount
```

**Comment by jeffcarp** [comment]
Thanks for taking the time to contribute this!! I've been meaning to check out Riot, it looks awesome. I'm ok with adding it to the table. I left a couple notes.

**Comment by crisward** [comment]
Made those changes. Many thanks.

**Comment by jeffcarp** [comment]
One problem I'm running into is that adding Riot makes the table so wide that the headers are now pushed way over. Thinking of how to deal with this...

screen shot 2016-06-04 at 2 22 53 pm

**Comment by jeffcarp** [comment]
Riot.js is in!!!!! Thank you! 🎉 

high five

**Comment by crisward** [comment]
Thanks!

**Pull request closed without merging**